src/Microsoft.Compatibility.HashCode/ref/Microsoft.Compatibility.HashCode.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
+  <PropertyGroup>
+    <ProjectGuid>{96AA2060-C846-4E56-9509-E8CB9C114C8F}</ProjectGuid>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release;uap-Debug;uap-Release</Configurations>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
+    <Compile Include="Microsoft.Compatibility.HashCode.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
+    <Compile Include="Microsoft.Compatibility.HashCode.Forwards.cs" />
+    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" Condition="'$(TargetGroup)' != 'netcoreapp2.1' AND '$(TargetGroup)' != 'netstandard2.1'" />
+    <Reference Include="System.Runtime" Condition="'$(TargetGroup)' == 'netcoreapp2.1' OR '$(TargetGroup)' == 'netstandard2.1'" />
```

Comment:
You can remove `netstandard2.1` here, everything should be present in netstandard.dll.